File: android/versioned-abis/expoview-abi48_0_0/src/main/java/abi48_0_0/expo/modules/gl/GLViewModule.kt
package abi48_0_0.expo.modules.gl

import abi48_0_0.expo.modules.kotlin.modules.Module
import abi48_0_0.expo.modules.kotlin.modules.ModuleDefinition

class GLViewModule : Module() {
  override fun definition() = ModuleDefinition {
    Name("ExponentGLView")

    View(GLView::class) {
      Events("onSurfaceCreate")
    }
  }
}

File: ios/versioned/sdk48/ExpoGL/ios/GLViewModule.swift
// Copyright 2022-present 650 Industries. All rights reserved.

import ABI48_0_0ExpoModulesCore

public final class GLViewModule: Module {
  public func definition() -> ModuleDefinition {
    Name("ExponentGLView")

    View(ABI48_0_0EXGLView.self) {
      Events("onSurfaceCreate")

      Prop("msaaSamples") { (view, msaaSamples: Int) in
        view.msaaSamples = msaaSamples
      }
    }
  }
}
